pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -182,10 +182,17 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
 		env = append(env, kapi.EnvVar{Name: "BUILD_LOGLEVEL", Value: buildLogLevel})
 	}
 
-	request := &buildapi.BuildRequest{
-		ObjectMeta: kapi.ObjectMeta{Name: name},
-	}
-	if len(env) > 0 {
+  buildCauses := []*buildapi.BuildTriggerCauses{}
+  request := &buildapi.BuildRequest{
+    ObjectMeta: kapi.ObjectMeta{Name: name},
+    BuildCauses: append(buildCauses,
+      &buildapi.BuildTriggerCauses{
```

Comment:
yes, it does make sense, what i didn't like was that we already have: `TriggeredByImage` that is not related to the trigger info in this PR.